tests/e2e_all/scripts/main.sh

```diff
@@ -110,19 +110,19 @@ export commitId
 remoteUsername=$(whoami)
 identityFilename="$HOME/.ssh/e2e_all.${commitId}"
 
-daemonStartWait=15
+daemonStartWait=20
```

Comment:
Increased wait time, as the c daemon synchronously puts username keys before starting monitor now.